modules/core/packer/consul/packer.json

```diff
@@ -72,7 +72,7 @@
     "provisioners": [
         {
             "type": "shell",
-            "inline": ["while ! [ -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting on cloud-init...'; sleep 2; done"]
+            "inline": ["timeout 60s bash -c \"while ! [ -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting on cloud-init...'; sleep 2; done\""]
```

Comment:
For testing, one can do on your own machine:

```bash
timeout 3s bash -c "while ! [ -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting on cloud-init...'; sleep 2; done"
```

Should return error code `124`.

Reply:
OK